interop/src/clients/corecrypto/android.rs

```diff
@@ -0,0 +1,434 @@
+use std::{
+    cell::{Cell, RefCell},
```

Comment:
`Cell` import should be guarded via `#[cfg(feature = "proteus")]` in order to silence the `unused import` warning.